### DISCUSSION TITLE
Other boards than arduino uno? D1 Based ESP8266 compatible?

### DISCUSSION THREAD

**Discussion by klsystem1** [yozik04/nibe#154] [Q&A]
Are there any other boards than arduino uno, for example nano or the D1 Based ESP8266 board compatible ? 
The ESP8266 has wifi on board, and I do only need an RS485 serial card to use.
thanks
Klaus

**Comment by elupus** [comment]
Not sure what you are asking. This project is a python project. It does not support any boards. It needs some gateway device or local rs485 connnection.

You can use an Arduino based gateway (available from openhab), an esphome based gateway (https://github.com/elupus/esphome-nibe) or a linux based gateway (available from openhab).

The esphome based is most up to date.
